doc/source/user_guide/sparse.rst

```diff
@@ -206,20 +283,18 @@ Pandas supports creating sparse dataframes directly from ``scipy.sparse`` matric
    sp_arr = csr_matrix(arr)
    sp_arr
 
-   sdf = pd.SparseDataFrame(sp_arr)
-   sdf
+   sdf = pd.DataFrame.sparse.from_spmatrix(sp_arr)
+   sdf.head()
+   sdf.dtypes
 
 All sparse formats are supported, but matrices that are not in :mod:`COOrdinate <scipy.sparse>` format will be converted, copying data as needed.
-To convert a ``SparseDataFrame`` back to sparse SciPy matrix in COO format, you can use the :meth:`SparseDataFrame.to_coo` method:
+To convert back to sparse SciPy matrix in COO format, you can use the :meth:`DataFrame.sparse.to_coo` method:
 
 .. ipython:: python
 
-   sdf.to_coo()
+   sdf.sparse.to_coo()
 
-SparseSeries
-~~~~~~~~~~~~
-
-A :meth:`SparseSeries.to_coo` method is implemented for transforming a ``SparseSeries`` indexed by a ``MultiIndex`` to a ``scipy.sparse.coo_matrix``.
+:meth:`Series.sparse.to_coo` is implemented for transforming a ``Series`` with sparse values indexed by a ``MultiIndex`` to a ``scipy.sparse.coo_matrix``.
```

Comment:
``:class:`MultiIndex` ``

Reply:
do we have the doc inventory for scipy? can you add a refernce to coo_matrix?

Reply:
We do have SciPy in our intersphinx.